testsuites/geosparql/src/main/java/org/eclipse/rdf4j/testsuite/query/algebra/geosparql/GeoSPARQLManifestTest.java

```diff
@@ -10,50 +10,14 @@
  *******************************************************************************/
 package org.eclipse.rdf4j.testsuite.query.algebra.geosparql;
 
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collection;
-import java.util.Deque;
-import java.util.List;
 
-import org.eclipse.rdf4j.query.Dataset;
 import org.eclipse.rdf4j.testsuite.query.parser.sparql.manifest.SPARQLQueryComplianceTest;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
+import org.junit.jupiter.api.DynamicTest;
 
-@RunWith(Parameterized.class)
 public abstract class GeoSPARQLManifestTest extends SPARQLQueryComplianceTest {
 
-	public GeoSPARQLManifestTest(String displayName, String testURI, String name, String queryFileURL,
-			String resultFileURL, Dataset dataset, boolean ordered, boolean laxCardinality) {
-		super(displayName, testURI, name, queryFileURL, resultFileURL, dataset, ordered, laxCardinality);
-	}
-
-	@Parameterized.Parameters(name = "{0}")
-	public static Collection<Object[]> data() {
-		return Arrays.asList(getTestData());
-	}
-
-	private static Object[][] getTestData() {
-
-		List<Object[]> tests = new ArrayList<>();
-
-		Deque<String> manifests = new ArrayDeque<>();
-		manifests.add(
-				GeoSPARQLManifestTest.class.getClassLoader()
-						.getResource("testcases-geosparql/functions/manifest.ttl")
-						.toExternalForm());
-		while (!manifests.isEmpty()) {
-			String pop = manifests.pop();
-			SPARQLQueryTestManifest manifest = new SPARQLQueryTestManifest(pop, null, false);
-			tests.addAll(manifest.getTests());
-			manifests.addAll(manifest.getSubManifests());
-		}
-
-		Object[][] result = new Object[tests.size()][6];
-		tests.toArray(result);
-
-		return result;
+	public Collection<DynamicTest> tests() {
```

Comment:
I believe that this needs the @TestFactory annotation.

Reply:
Probably applies to other similar methods too.

Reply:
I think it is actually an error in the manifest file itself. There is a query that looks for `mf:include` but the manifest file for geosparql uses `mf:entities` and even a different IRI prefix for `mf:`. Let me have a look.

Reply:
Looking at a current develop integration test job I think these tests are not run at this time either...
```
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO] 
[INFO] 
[INFO] --- maven-failsafe-plugin:3.1.2:verify (verify) @ rdf4j-geosparql-compliance ---
```

Reply:
Reenabled these tests. Lucky they all pass :)

Reply:
There are several other similar methods that were missing the @TestFactory annotation. Did you look over the rest of the code?